src/ocamlDep.re

```diff
@@ -19,16 +19,18 @@ let ocamlDep source::source target::target => {
   let flag = isInterface source ? "-intf" : "-impl";
   let ppx = target.engine == "bucklescript" ? "-ppx bsppx.exe" : "";
   let extraFlags = target.flags.dep;
+  let envvars = target.flags.envvars;
   /* betterError doesn't support bucklescript yet */
-  let berror = target.engine == "bucklescript" ? "" : "| berror";
+  let berror = target.engine == "bucklescript" ? "" : "2>&1| berror";
```

Comment:
Oh, another thing I realized is that there was no way to change `berror` to `refmterr`! I think we should add a config field, don't you? (Just to put the name of the binary `berror`/`refmterr`?

(This is totally unrelated to this diff).